botorch/models/approximate_gp.py

```diff
@@ -417,131 +448,3 @@ def init_inducing_points(
             var_strat.variational_params_initialized.fill_(0)
 
         return inducing_points
-
-
-def _select_inducing_points(
```

Comment:
Deleting `_select_inducing_points` is creating an import error for a downstream library ([aepsych](https://github.com/facebookresearch/aepsych)). Can `_select_inducing_points` either call one of the new functions or raise a deprecation warning?

Reply:
I think we probably don't have to get too crazy in terms of deprecations here since this is a "private" method - @crasanders could we just update aepsych to use the new and shiny initialization strategies here?

Reply:
What did you two decide?

Reply:
We'll fix the aepsych issue ourselves and then will merge those changes together. @esantorella could you take care of this, please?

Reply:
Yup I'll put in a PR to aepsych